.github/workflows/integration.yml

```diff
@@ -44,8 +45,11 @@ jobs:
       - name: Pull images
         run: docker-compose -f docker-compose-avro.yml pull --ignore-pull-failures
 
-      - name: TiKV version
-        run: docker run pingcap/tikv:release-4.0-nightly -V
+      - name: TiDB version
+        run: |
+          docker run pingcap/tidb:nightly -V
+          docker run pingcap/tikv:nightly -V
+          docker run pingcap/pd:nightly -V
```

Comment:
Image tag`v5.2.0-nightly` is not available in docker hub currently, which will be uploaded after `5.2.0` is released. Since the integration test is not enabled in release branch, it is safe to change all tags to `v5.2.0-nightly` now.